GOVERNANCE.adoc

```diff
@@ -0,0 +1,71 @@
+This document describes the governance rules of the ServiceTalk project/organization. It is meant to be followed by all
+the repositories / sub-projects in the organization.
+
+== Contributions
+Everyone is welcome to contribute to ServiceTalk. Contributions aren't limited to submitting pull requests there are
+many different ways for you to get involved, including answering questions and reporting or triaging bugs. No matter how
+you want to get involved, we ask that you first learn what’s expected of anyone who participates in the project by
+reading the xref:CONTRIBUTING.adoc[Contributor Guidelines].
+
+== Decision Making and Voting
```

Comment:
isn't `Decision Making` one word ? Or at least shouldn't it be `Decision-Making` ? That said I am not a native speaker so I may be wrong :)

Reply:
I'll add a hyphen.